common/script/fns/index.js

```diff
@@ -20,6 +21,7 @@ module.exports = {
   randomDrop,
   autoAllocate,
   updateStats,
+  statsComputed,
```

Comment:
This is weird. We should look in the code and see where it's being used. If it doesn't need to be on the fns object, it shouldn't be.